packages/openjdk17.rb

```diff
@@ -0,0 +1,43 @@
+require 'package'
+
+class Openjdk17 < Package
+  description 'The JDK is a development environment for building applications, applets, and components using the Java programming language.'
+  homepage 'https://openjdk.org/'
+  version '17.36.19'
```

Comment:
this makes more sense to me than marking it as 17.0.4.1 in case, say, they hypothetically screwed up their 17.0.4.1 upload and had to release 17.36.20 as a hotfix

Reply:
What is the version when you type `java -version`?

Reply:
I'm pretty sure it would be 17.0.4.1, although I haven't gotten it successfully installed yet

Reply:
Shouldn't we list the version as what the version is? Last time I checked chromebrew will "update" if it detects any version change, so if we need to go back a version then the update will process that fine.

Reply:
Sounds good, I'll change the version name once I get around to figuring out why it won't install properly